src/components/Persona/Persona.tsx

```diff
@@ -23,19 +23,29 @@ export class Persona extends React.Component<IPersonaProps, any> {
   };
 
   public render() {
-    let { className, size, imageUrl, imageInitials, initialsColor, presence, primaryText, secondaryText, tertiaryText, optionalText, hidePersonaDetails } = this.props;
+    let {
+      className,
+      size,
+      imageUrl,
+      imageInitials,
+      initialsColor,
+      presence,
+      primaryText,
+      secondaryText,
+      tertiaryText,
+      optionalText,
+      hidePersonaDetails
+    } = this.props;
 
     return (
-      <div { ... this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
+      <div { ...this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
```

Comment:
sorry David, I shouldn't make changes to the Persona.tsx ...sorry, I picked up my old changes, I should only have changes for the Panel.css and Callout.css